src/loci/formats/in/ZarrReader.java

```diff
@@ -627,11 +627,19 @@ private void parseOmeroMetadata(String root, String key) throws IOException, For
       }
     }
   }
-  
+
+  private Double getDouble(Map<String, Object> src, String key) {
+    Number val = (Number) src.get(key);
+    if (val == null) {
```

Comment:
Potentially an use case for `DataTools.parseDouble()` here as it shoudl include most of the nullity checks as well as extra handling for the decimal separator

Reply:
Except there's no string parsing here, right? That all happens down at the JSON layer.

Reply:
Correct. The above assumed a `Map<String, String>` source. Forget about me ;)